src/models/room.ts

```diff
@@ -186,7 +192,8 @@ export class Room extends ReadReceipt<EmittedEvents, RoomEventHandlerMap> {
     public readonly reEmitter: TypedReEmitter<EmittedEvents, RoomEventHandlerMap>;
     private txnToEvent: Record<string, MatrixEvent> = {}; // Pending in-flight requests { string: MatrixEvent }
     private notificationCounts: NotificationCount = {};
-    private threadNotifications: Map<string, NotificationCount> = new Map();
+    private readonly threadNotifications: Map<string, NotificationCount> = new Map();
```

Comment:
Shorthand would be this fwiw:
```suggestion
    private readonly threadNotifications = new Map<string, NotificationCount>();
```

Reply:
This kind of style comments should be enforced as part of the code style and not be part of reviews IMO.

Reply:
This was a suggestion, not a requirement.